control-board/src/robot_state.rs

```diff
@@ -44,6 +45,7 @@ impl SharedRobotState {
             radio_inop: AtomicBool::new(true),
             imu_inop: AtomicBool::new(true),
             kicker_inop: AtomicBool::new(true),
+            power_inop: AtomicBool::new(true),
```

Comment:
Might have missed the usage but is the flag cleared?

Reply:
nope good catch, pushed a fix. If we don't get a packet for 500ms from power, it becomes inop, if we do its op